### PR TITLE
docs: clarify other important improvement regarding disambiguation of…

### PR DESCRIPTION
**Pull request by claymcleod**
… read evidence

### CODE REVIEW

**Comment by claymcleod** [comment]
@a-frantz please merge this in if you are happy.

**Comment by a-frantz** [comment]
@claymcleod I was right, my GFF reimplementation made the the GTF preparation steps unnecessary. Any valid GFF/GTF should be supported now (including a known-malformed ENSEMBL release). I updated the docs to reflect that, and asked users to report any unsupported GFFs.

File: docs/subcommands/strandedness.md [hunk 1]
```diff
@@ -1,98 +1,40 @@
...
-        key = groups.group(1)                    
-        value = groups.group(2).replace("\"", "").replace(" ", ",")
-        attrs_post.append(key + "=" + value)
-
-    print(";".join(attrs_post))
-END
-) | sed 's/^chrM/chrMT/g' | sed 's/^chr//g' | bgzip > gencode.v32lift37.annotation.gtf.gz
-tabix -p gff gencode.v32lift37.annotation.gtf.gz
-```
+Any valid GTF or GFF file should be compatible with `ngsderive`. The strandedness check requires random access of the gene model, which means the file must be sorted and tabixed. If you supply an unsorted or untabixed GTF/GFF, we will create one for you using the supplied file. If you encounter any errors related to your gene model choice, please let us know using GitHub Issues.
```
Comment (claymcleod) [inline docs/subcommands/strandedness.md:20]: Link to GH issues?

**Comment by claymcleod** [comment]
Seems okay to me.

**Pull request merged**